common/src/main/java/tachyon/conf/TachyonConf.java

```diff
@@ -308,6 +316,10 @@ public long getBytes(String key, long defaultValue) {
     }
   }
 
+  public long getBytes(String key) {
+    return getBytes(key, 0L);
+  }
+
```

Comment:
+1. You can throw some Exception with message like invalid configuration and etc. We shall not "help" users to decide the default value. To finish this PR, you probably need to add some properties in `tachyon-default.properties` if there are properties set in code but can not be found in `tachyon-default.properties` (most likely our fault to update `tachyon-default.properties`)